po/cs.po

```diff
@@ -43,70 +43,72 @@ msgid ""
 "Axolotl encountered an error communicating with Signal servers. Please try "
 "again."
 msgstr ""
+"Axolotl má problém v komunikaci se serverem Signálu. "
+"Zkuste to prosím znovu."
 
 #: src/pages/Contacts.vue:3
 msgid "Can't change contact list: {{error}}"
-msgstr ""
+msgstr "Chyba při změně v seznamu kontaktů: {{error}}"
 
 #: src/components/ErrorModal.vue:18 src/components/IdentityModal.vue:21
 #: src/components/ImportUnavailableModal.vue:15
 #: src/components/StartChatModal.vue:13
 msgid "Close"
-msgstr ""
+msgstr "Zavřít"
 
 #: src/components/ConfirmationModal.vue:13
 msgid "Confirm"
-msgstr ""
+msgstr "Potvrdit"
 
 #: src/components/Header.vue:73
 msgid "Connect with Signal"
-msgstr "Připojení pomocí Signál"
+msgstr "Připojit se k Signálu"
 
 #: src/components/AddContactModal.vue:5 src/components/EditContactModal.vue:5
 msgid "Contact"
 msgstr "Kontakt"
 
 #: src/pages/Contacts.vue:16
 msgid "Contact list is empty..."
-msgstr ""
+msgstr "Seznam kontaktů je prázdný..."
 
 #: src/components/Header.vue:146
 #, fuzzy
```

Comment:
Hey thanks for the translation please can you remove all the `#, fuzzy` because they disable the sentence from being used. Otherwise it looks quite good :)

Reply:
But only for the parts, where a source file is given like `#: src/components/Header.vue:27 src/pages/ChatList.vue:25`